### PR TITLE
[skip ci] Update docs

### PR DESCRIPTION
**Pull request by jasmezz**
Updating documentation, including feedback from Martin.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jasmezz** [comment]
[skip ci] comes at the beginning, did not know that, sorry waste of resources :cry:

**Comment by jasmezz** [comment]
Renaming doesn't stop tests, of course...

File: README.md [hunk 1]
```diff
@@ -12,22 +12,24 @@
...
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers and supports conda package manager making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per tool which makes it much easier to maintain and update software dependencies. Where possible, these tools have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community.
```
Comment (jfy133) [inline README.md:17]: Please check this matches the template! https://github.com/nf-core/tools/tree/master/nf_core/pipeline-template

File: README.md [hunk 2]
```diff
@@ -12,22 +12,24 @@
...
 
-On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources.The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/funcscan/results).
+<!-- TODO nf-core: Add full-sized test dataset and amend the paragraph below if applicable -->
```
Comment (jfy133) [inline README.md:19]: Can remove this now, no?

File: README.md [hunk 3]
```diff
@@ -12,22 +12,24 @@
...
 
-The nf-core/funcscan AWS full test dataset are contigs generated by the MGnify service from the ENA. We used contigs generated from assemblies of chicken cecum shotgun metagenomes (study accession: MGYS00005631).
+On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/funcscan/results).
```
Comment (jfy133) [inline README.md:21]: Why have you remove the extra line about the contents of what we use for the full test data?

File: docs/usage.md [hunk 4]
```diff
@@ -332,16 +333,16 @@ Tip: you can replicate the issue by changing to the process work dir and enterin
...
+A first step to bypass this error, you could try to increase the amount of CPUs, memory, and time for the whole pipeline by increasing the resource for the parameters `--max_cpus`, `--max_memory`, and `--max_time`. Based on the error above, you have to increase the amount of memory. Therefore, you can go to the [parameter documentation of funcscan](https://nf-co.re/funcscan/dev/parameters) and scroll down to the `show hidden parameter` button to get the default value for `--max_memory`. If the default is 128 GB, you can try to run your pipeline again with for example `--max_memory 200GB -resume` to skip all process, that were already calculated. If you can not increase the resource of the complete pipeline, you can try to adapt the resource for a single process as mentioned below.
```
Comment (jfy133) [inline docs/usage.md:336]: Double check there is only minimal changes from the template see above

File: docs/usage.md [hunk 5]
```diff
@@ -332,16 +333,16 @@ Tip: you can replicate the issue by changing to the process work dir and enterin
...
+The [Nextflow `label`](https://www.nextflow.io/docs/latest/process.html#label) directive allows us to organise workflow processes in separate groups which can be referenced in a configuration file to select and configure subsets of processes having similar computing requirements.
 The default values for the `process_high` label are set in the pipeline's [`base.config`](https://github.com/nf-core/rnaseq/blob/4c27ef5610c87db00c3c5a3eed10b1d161abf575/conf/base.config#L33-L37) which in this case is defined as 72GB.
-Providing you haven't set any other standard nf-core parameters to **cap** the [maximum resources](https://nf-co.re/usage/configuration#max-resources) used by the pipeline then we can try and bypass the `STAR_ALIGN` process failure by creating a custom config file that sets at least 72GB of memory, in this case increased to 100GB.
+Providing you haven't set any other standard nf-core parameters to **cap** the [maximum resources](https://nf-co.re/usage/configuration#max-resources) used by the pipeline, then you can try to bypass the `STAR_ALIGN` process failure by creating a custom config file that sets at least 72GB of memory, in this case increased to 100GB.
```
Comment (jfy133) [inline docs/usage.md:345]: These I would not change, again these are form the template

File: docs/usage.md [hunk 6]
```diff
@@ -398,7 +399,7 @@ The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementatio
...
+In most cases, you will only need to create a custom config as a one-off but if you and others within your organisation are likely to be running nf-core pipelines regularly and need to use the same settings regularly it may be a good idea to request that your custom config file is uploaded to the `nf-core/configs` git repository. Before you do this please test that the config file works with your pipeline of choice using the `-c` parameter. You can then create a pull request to the `nf-core/configs` repository with the addition of your config file, associated documentation file (see examples in [`nf-core/configs/docs`](https://github.com/nf-core/configs/tree/master/docs)), and amending [`nfcore_custom.config`](https://github.com/nf-core/configs/blob/master/nfcore_custom.config) to include your custom profile.
```
Comment (jfy133) [inline docs/usage.md:402]: Don't change this, again template!

File: docs/usage.md [hunk 7]
```diff
@@ -419,7 +420,7 @@ Nextflow handles job submissions and supervises the running jobs. The Nextflow p
 The Nextflow `-bg` flag launches Nextflow in the background, detached from your terminal so that the workflow does not stop if you log out of your session. The logs are saved to a file.
 
 Alternatively, you can use `screen` / `tmux` or similar tool to create a detached session which you can log back into at a later time.
-Some HPC setups also allow you to run nextflow within a cluster job submitted your job scheduler (from where it submits more jobs).
+Some HPC setups also allow you to run nextflow within a cluster job submitted by your job scheduler (from where it submits more jobs).
```
Comment (jfy133) [inline docs/usage.md:423]: TEmplate again!

**Review by jfy133 — COMMENTED** [review]
Ok, everywhere that I've said 'don't change it's from the template', we don't want to change these here because it'll make future template merges much more difficult.

Instead, you should actually update the template itself via a PR from a fork of `nf-core/tools` into the dev branch of nf-core/tools - as actually the changes you suggest I do actually agree with :) Then next template merge we will have the better prhasing

**Comment by jfy133** [comment]
> agraph "Advanced option on process level" there is some reference to rnaseq pipeline. Need to take a look at that again.

No it can be anywhere in the _commit_ message, not the PR title (I beleve)

File: docs/usage.md [hunk 8]
```diff
@@ -111,19 +113,19 @@ And then passed to the pipeline with
...
 ### hmmsearch
 
-nf-core/funcscan allows screening of sequences for various natural product types via hidden markov models with HMMSearch.
+nf-core/funcscan allows screening of sequences for functional genes associated with various natural product types via Hidden Markov Models (HMMs) using HMMSearch.
 
-This requires supplying a list of HMM model files ending in `.hmm`, that have models for the particular molecule(s) you are interested in.
+This requires supplying a list of HMM files ending in `.hmm`, that have models for the particular molecule(s) or BGCs you are interested in.
 
-You can download such files from places such as [https://pfam.xfam.org/](https://pfam.xfam.org/) for antimicrobial peptides (AMP), or the antiSMASH github repository for [biosynthetic gene cluster](https://github.com/antismash/antismash/tree/master/antismash/detection/hmm_detection/data) related HMM models
+You can download these files from places such as [PFAM](http://pfam-legacy.xfam.org/) for antimicrobial peptides (AMP), or the antiSMASH GitHub repository for [biosynthetic gene cluster](https://github.com/antismash/antismash/tree/master/antismash/detection/hmm_detection/data) related HMMs.
```
[the remark this answers is not in the document]
Comment (jasmezz) [reply docs/usage.md:120]: I just wanted to say, that I already searched the site and didn't find any HMMs. I just searched again. Found them after 1 min :roll_eyes:

[the remark this answers is not in the document]
Comment (jfy133) [reply docs/usage.md]: Long day?

**Pull request merged**